CS3247-Game-Development-Team-6/Doodles
# scaling screen and button disable

closes #330 
closes #333 

to test:
- check if button is disabled before selecting a spell during spell selection
![image](https://user-images.githubusercontent.com/23572486/197329891-be8801e9-40a1-41e9-98be-5e7b3782edd1.png)
- check if pause screen is centered and scaling when changing different aspect ratios/resolutions
![image](https://user-images.githubusercontent.com/23572486/197329914-50ec1090-2973-4809-9065-d25859fdcceb.png)
